servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcExceptionMapperServiceFilter.java

```diff
@@ -84,9 +84,11 @@ private static StreamingHttpResponse convertToGrpcErrorResponse(final String wha
                 ctx.executionContext().bufferAllocator(), null);
         final CharSequence codeValue = response.headers().get(GRPC_STATUS);
         assert codeValue != null;
-        LOGGER.error("Unexpected exception during a {} processing for connection={}, request='{} {} {}' was mapped " +
-                        "to grpc-status: {} ({})", what, ctx, request.method(), request.requestTarget(),
-                request.version(), codeValue, fromCodeValue(codeValue), cause);
+        if (LOGGER.isInfoEnabled()) {
```

Comment:
@idelpivnitskiy - note this happens for all timeouts, logging at error for timeouts seems too high, wdyt?

Reply:
Can we log at different levels for timeout exceptions and keep `error` for everything else?

Reply:
yes let me exclude this log for timeout exception ... folks can get visibility via Observer if required.